presto-memsql/src/test/java/io/prestosql/plugin/memsql/TestMemSqlTypeMapping.java

```diff
@@ -300,6 +309,31 @@ public void testTimestamp()
         throw new SkipException("TODO");
     }
 
+    @Test
+    public void testJson()
+    {
+        jsonTestCases(jsonDataType(value -> "JSON " + formatStringLiteral(value)))
+                .execute(getQueryRunner(), prestoCreateAsSelect("presto_test_json"));
+        // TODO MemSQL doesn't support CAST to JSON but accepts string literals as JSON values
```

Comment:
CAST to JSON doesn't exist. There's a TO_JSON function but that parses strings into JSON values. ie. strings get converted into double quoted JSON strings. Passing strings that are valid JSON seem to get parsed into JSON during INSERT.

Reply:
Having comment is good, but this shouldn't be a TODO, because that's nor our problem.